flink-formats/flink-sql-avro-confluent-registry/src/main/resources/META-INF/NOTICE

```diff
@@ -10,10 +10,12 @@ This project bundles the following dependencies under the Apache Software Licens
 - com.fasterxml.jackson.core:jackson-core:2.15.3
 - com.fasterxml.jackson.core:jackson-databind:2.15.3
 - com.google.guava:guava:32.0.1-jre
+- commons-io:commons-io:2.11.0
 - io.confluent:common-utils:7.5.3
 - io.confluent:kafka-schema-registry-client:7.5.3
 - org.apache.avro:avro:1.11.3
-- org.apache.commons:commons-compress:1.24.0
+- org.apache.commons:commons-compress:1.26.0
```

Comment:
In the compilation report, we can find the following error report. The report says this issue must be fixed, but I'm not sure if we should fix it in this PR because it doesn't seem to be related.

```
09:19:48,111 INFO  org.apache.flink.tools.ci.licensecheck.NoticeFileChecker     [] - Problems were detected for a NOTICE file.
	flink-sql-avro-confluent-registry:
		 These issue are legally problematic and MUST be fixed: 
			Dependency commons-io:commons-io:2.11.0 is not listed.
			Dependency org.apache.commons:commons-lang3:3.12.0 is not listed.
```

Reply:
I would assume these are pulled in as transitive dependencies via the upgrade, thanks for adding them.